src/internal/helm/chart.go

```diff
@@ -38,7 +38,7 @@ func InstallOrUpgradeChart(options ChartOptions) types.ConnectStrings {
 		options.Chart.Name,
 		options.Chart.Version,
 		fromMessage)
-	defer spinner.Stop()
+	defer spinner.Success()
```

Comment:
If this function breaks at line 83 it would erroneously turn the spinner into a success.

Reply:
Yeah we need to talk more about failure behavior, should we just abort the whole thing if something fails?  Not totally sure, but I did miss that failure case, so I'll change it to existing behavior for now.